easybuild/easyblocks/h/hypre.py

```diff
@@ -40,20 +42,20 @@ def configure_step(self):
 
         self.cfg.update('configopts', '--with-MPI-include=%s' % os.getenv('MPI_INC_DIR'))
 
-        for dep in ["BLAS", "LAPACK"]:
-            libs = ' '.join(os.getenv('%s_STATIC_LIBS' % dep).split(','))
-            self.cfg.update('configopts', '--with-%s-libs="%s"' % (dep.lower(), libs))
-            self.cfg.update('configopts', '--with-%s-lib-dirs="%s"' % (dep.lower(),
-                                                                      os.getenv('%s_LIB_DIR' % dep)))
+        # Only supports external libraries when building a shared library.
+        self.cfg.update('configopts', '--enable-shared')
+
+        # While there are a --with-{blas|lapack}-libs flag, it's not useable, because of how Hypre treats it.
+        # We need to patch the code anyway to prevent it from building it's own blas packages.
```

Comment:
typo: building its own BLAS packages

Reply:
@Micket ping on fixing the typo?

Reply:
@Micket Typo is still there: `it's` should be `its` ;)